test/regressions/.mocharc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
+  extension: ['js', 'ts', 'tsx'],
```

Comment:
Do we need to duplicate it if this entry is present in `setupBabel`? 🤔

Reply:
yeah, seems to throw a bunch of errors otherwise 🙃

Reply:
Gotcha. 👌